### DISCUSSION TITLE
Where can I find changelog for @vercel/og ?

### DISCUSSION THREAD

**Discussion by jinsley8** [vercel/community#2068] [Help] [answered]
### Problem Description

Where can I find the changelog for `@vercel/og`?

**Comment by amyegan** [comment, marked as answer]
You can find the latest info about `@vercel/og` in the [Open Graph (OG) Image Generation](https://vercel.com/docs/concepts/functions/edge-functions/og-image-generation#) documentation and the [Satori](https://github.com/vercel/satori) repo. You can also watch the [Vercel Changelog](https://vercel.com/changelog) for future announcements
